docs/Overview.md

```diff
@@ -11,6 +11,10 @@ Draft.js is a framework for building rich text editors in React, powered by an i
 
 Draft.js makes it easy to build any type of rich text input, whether you're just looking to support a few inline text styles or building a complex text editor for composing long-form articles.
 
+We first introduced Draft.js at the [React.js Conf](http://conf.reactjs.com/) in february:
```

Comment:
Draft.js was introduced at [React.js Conf](http://conf.reactjs.com/) in February 2016.